hypervisor/src/lib.rs

```diff
@@ -67,14 +67,42 @@ pub enum HypervisorType {
     Mshv,
 }
 
-pub fn new() -> std::result::Result<Arc<dyn Hypervisor>, HypervisorError> {
-    #[cfg(feature = "kvm")]
-    let hv = kvm::KvmHypervisor::new()?;
+#[derive(Debug, Eq, PartialEq)]
+pub enum HypervisorSelection {
+    AUTO,
```

Comment:
This enum looks wrong. Is there a clippy override somewhere in this file or crate?

Reply:
Do you mean it should be `Auto` instead?

Yes there is an override in the hypervisor crate.

Reply:
I meant the capitalisation. It should be "Auto, Kvm, Mshv" for consistency with the other enums. I thought clippy would be angry with fully capitalised enum variants.